vulyk/control.py

```diff
@@ -197,7 +197,6 @@ def project_init(allowed_types: list[str]) -> None:
 @cli.group("stats")
 def stats() -> None:
     """Commands to show some stats."""
-    pass
```

Comment:
Explicit is better than implicit?

Reply:
It gives similar warning in all the linters out there: https://docs.astral.sh/ruff/rules/unnecessary-placeholder/